src/App.jsx

```diff
@@ -10,6 +10,7 @@ const App = () => (
     <Switch>
       <Route path="/" exact>
         <HomepageImage />
+        <RestaurantList />;
```

Comment:
Usually, we keep one Component per route. So how about creating a parent component like `HomePage` and put these two inside that component?